Prolific/Prolific/Utils/User Engagement Manager/UserEngagementManager.h

```diff
@@ -8,10 +8,24 @@
 
 #import <Foundation/Foundation.h>
 
+#import "User.h"
+
 NS_ASSUME_NONNULL_BEGIN
 
 @interface UserEngagementManager : NSObject
 
+typedef NS_ENUM(NSInteger, UserEngagementType) {
```

Comment:
Enums! nice job. You even got the annoying pattern on putting the whole enum name in each case :)